tests/ExerciseConfig/Validation/ExerciseConfigValidator.phpt

```diff
@@ -1,6 +1,6 @@
 <?php
 
-include '../../bootstrap.php';
+$container = include '../../bootstrap.php';
```

Comment:
Seems to be not used in here?

Reply:
check out line #227

Reply:
yes you are using it as a parameter for test class, but in the test class it is not used, or IntelliJ is trying to tell me it is not...

Reply:
My IntelliJ seems OK with it.

Reply:
Sorry poor choice of words. The constructor parameter is assigned to internal $this->container and this isntance container is not used anywhere else then in constructor. But this is only small detail.